### PR TITLE
Doc: Updating Docs for Kubernetes deployment

### PR DESCRIPTION
**Pull request by Prajwal214**
### Describe your changes:

Fixes <issue-number>

Updating Docs for Kubernetes deployment

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-docs/content/partials/v1.4/deployment/faqs-eks.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# FAQs
```
Comment (akash-jain-10) [inline openmetadata-docs/content/partials/v1.4/deployment/faqs-eks.md:1]: Let's remove this heading!

File: openmetadata-docs/content/partials/v1.5/deployment/faqs-eks.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# FAQs
```
Comment (akash-jain-10) [inline openmetadata-docs/content/partials/v1.5/deployment/faqs-eks.md:1]: Let's remove this heading!

**Review by akash-jain-10 — CHANGES_REQUESTED** [review]

File: openmetadata-docs/content/partials/v1.4/deployment/faqs-eks.md (hunk 1 above)
Comment (Prajwal214) [reply openmetadata-docs/content/partials/v1.4/deployment/faqs-eks.md]: I have removed it

File: openmetadata-docs/content/partials/v1.5/deployment/faqs-eks.md (hunk 2 above)
Comment (Prajwal214) [reply openmetadata-docs/content/partials/v1.5/deployment/faqs-eks.md]: I have removed it

**Review by akash-jain-10 — APPROVED** [review]
LGTM 🚀

**Pull request merged**